src/core/meshcop/border_agent.cpp

```diff
@@ -610,42 +610,47 @@ Error BorderAgent::ForwardToLeader(const Coap::Message &aMessage, const Ip6::Mes
     return error;
 }
 
-void BorderAgent::HandleTmfDatasetGet(Coap::Message          &aMessage,
-                                      const Ip6::MessageInfo &aMessageInfo,
-                                      Dataset::Type           aType)
+void BorderAgent::HandleTmfDatasetGet(Coap::Message &aMessage, const Ip6::MessageInfo &aMessageInfo, Uri aUri)
```

Comment:
Suggest maybe renaming this to `HandleTmfGet()` since it can now get more than Active/Pending Datasets?

Reply:
It's fine to keep the name as is. We may not always use the same names as in the spec in all cases. The spec calls it "Commissioner Dataset," but within OT we use "Commissioner Data" (e.g. `FindCommissioningData()`), which is shorter but similar.